src/core/dynamic_batch_scheduler.h

```diff
@@ -115,9 +134,10 @@ class DynamicBatchScheduler : public Scheduler {
   std::mutex mu_;
   std::condition_variable cv_;
 
-  // Queue holding inference requests for the model represented by
-  // this scheduler.
-  std::deque<Scheduler::Payload> queue_;
+  // Map from priority level to queue holding inference requests for the model
+  // represented by this scheduler. Priority level 0 is reserved for default
```

Comment:
"Priority level 0...". Do you mean that if priority queues are not supported by the scheduler than priority zero entry is used as the single queue?

Reply:
Yes